Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -286,6 +286,21 @@ RDKIT_FILEPARSERS_EXPORT RWMol *Mol2BlockToMol(
     const std::string &molBlock, bool sanitize = true, bool removeHs = true,
     Mol2Type variant = CORINA, bool cleanupSubstructures = true);
 
+RDKIT_FILEPARSERS_EXPORT Atom *ParseXYZFileAtomLine(const std::string &atomLine,
```

Comment:
Any reason not to make these unique_ptrs?

Reply:
Actually, this doesn't need to be in the public API at all